recipes/debugging/vs-code-debug.md

```diff
@@ -44,21 +61,151 @@ Debugging an Amber project on VSCode with GDB and [Native Debug](https://github.
       "name": "Debug",
       "type": "gdb",
       "request": "launch",
-      "target": "./app",
+      "target": "./bin/${workspaceFolderBasename}",
       "cwd": "${workspaceRoot}",
-      "preLaunchTask": "amber"
+      "preLaunchTask": "Compile"
     }
   ]
 }
 ```
 
-**3. Then hit the green play button**
+#### Using LLDB
 
-![green button](https://camo.githubusercontent.com/30adba87add4770abf2c3982206748123f8a2c6e/687474703a2f2f692e696d6775722e636f6d2f6d674b41366d782e706e67)
+```json
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Debug",
+      "type": "lldb-mi",
+      "request": "launch",
+      "target": "./bin/${workspaceFolderBasename}",
+      "cwd": "${workspaceRoot}",
+      "preLaunchTask": "Compile"
+    }
+  ]
+}
+```
+
+### 3. Then hit the DEBUG green play button
+
+![debugging](https://i.imgur.com/GsGT1h0.png)
+
+## Trips and Tricks for debugging Crystal applications
 
-![change variable value](https://camo.githubusercontent.com/c5a551366c3eb2464c920bf3f95e8cdfb97ad827/687474703a2f2f692e696d6775722e636f6d2f6b506b546e75442e706e67)
+> DISCLAIMER: LLDB doesn't show variables data for crystal applications yet, see [issue #4457](https://github.com/crystal-lang/crystal/issues/4457).
 
-[Native Debug](https://github.com/WebFreak001/code-debug) allows to set breakpoints, watch variables and execute GDB commands inside VSCode.
+Actually debugging Crystal application is not fully supported yet. However you can use some techniques to improve your debugging experience.
```

Comment:
Fully debugging Crystal applications is not supported yet. You can use some of the techniques below to improve the debugging experience.